src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator
+
+import calculator.type.Operator
+import calculator.validator.OperationValidator
+
+class Calculator {
+    fun calculate(expression: String): Int {
+        val expressionList = extractExpressionList(expression)
+
+        var result = expressionList[0].toInt()
+        for (index in 1 until expressionList.size step 2) {
+            val operator = Operator.of(expressionList[index])
+            if (operator != Operator.NONE) {
+                val rightOperand = expressionList.getOrNull(index + 1)?.toIntOrNull() ?: continue
+                result = performOperation(result, rightOperand, operator)
+            }
+        }
+
+        return result
+    }
+
+    private fun extractExpressionList(expression: String): List<String> {
+        val expressionList = expression.split(" ")
+        if (OperationValidator.hasValidSize(expressionList).not()) throw IllegalArgumentException("잘못된 입력입니다.")
```

Comment:
kotlin이 제공하는 `require` 함수를 활용해볼 수 있을 것 같아요. 😃

```suggestion
        require(OperationValidator.hasValidSize(expressionList)) { "잘못된 입력입니다." }
```